openshift-ci/Dockerfile.tools

```diff
@@ -0,0 +1,59 @@
+FROM centos:7 as build-tools
+
+LABEL maintainer "Devtools <devtools@redhat.com>"
+LABEL author "Devtools <devtools@redhat.com>"
+
+ENV LANG=en_US.utf8 \
+    GOPATH=/tmp/go \
+    GOCACHE=/tmp/.cache \
+    PATH=$PATH:$GOPATH/bin \
+    GIT_COMMITTER_NAME=devtools \
+    GIT_COMMITTER_EMAIL=devtools@redhat.com \
+    OPERATOR_SDK_VERSION=v0.10.0
+
+ARG GO_PACKAGE_PATH=github.com/codeready-toolchain/registration-service
+
+RUN yum install epel-release -y \
+    && yum install  https://centos7.iuscommunity.org/ius-release.rpm -y \
+    && yum install --enablerepo=centosplus install -y --quiet \
+    findutils \
+    git2u-all \
+    golang \
+    make \
+    procps-ng \
+    tar \
+    wget \
+    which \
+    bc \
+    kubectl \
+    yamllint \
+    python36-virtualenv \
+    jq \
+    go-bindata \
+    && yum clean all
+
+# download, verify and install openshift client tools (oc and kubectl)
+WORKDIR /tmp
+RUN curl -L -s https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -o openshift-origin-client-tools.tar.gz \
+    && echo "4b0f07428ba854174c58d2e38287e5402964c9a9355f6c359d1242efd0990da3 openshift-origin-client-tools.tar.gz" > openshift-origin-client-tools.sha256 \
+    && sha256sum -c openshift-origin-client-tools.sha256 \
+    && tar xzf openshift-origin-client-tools.tar.gz \
+    && mv /tmp/openshift-origin-client-tools-*/oc /usr/bin/oc \
+    && mv /tmp/openshift-origin-client-tools-*/kubectl /usr/bin/kubectl \
+    && rm -rf ./openshift* \
+    && oc version
+
+# download, verify and install operator-sdk
+RUN curl -L -s https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu -o operator-sdk \
```

Comment:
Do we need this for registration service?

Reply:
Not right now. But it’s preparation for running e2e tests in openshif-ci. We will need operator-sdk for that.